### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by spydaspider**
# Mobile Menu for my Portfolio
# On small screens like mobile, there is a mobile menu that displays menu of the links to navigate the pages.  
- [ ] Bug fix (unexpected generic font-family).
- [ ] New feature ( font-family; Poppins, Arial, Helvetica, sans-serif;)
- [ ] New files (.eslintrc.json)
  - [ ] Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix  after changing the font.
# Checklist
 [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,15 +11,21 @@
...
+        <ul class = "nav-menu">
+          <li class = "nav-item"><a href = "#headline">Portfolio</a></li>
+          <li class = "nav-item"><a href = "#section-5">About</a></li>
+          <li class = "nav-item"><a href = "#section-7">Contact</a></li>
+        </ul>
+        <div class="x-icon x-style">
+          <span class = "bar-x"></span>
+          <span class = "bar-x"></span>
+          <span class = "bar-x"></span>
+
```
Comment (muneebulrehman) [inline index.html]: Kindly follow the margin/paddings as per the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: index.html [hunk 2]
```diff
@@ -11,15 +11,21 @@
...
       <nav class="nav-container">
         <div class="logo-text"><a href="#">Spyda</a></div>
-        <div class="right-logo">
-          <img
-            src="images/menu-icon.png"
-            alt="Menu Icon for toggle on small screens"
-          />
+      <div class = "mobile-menu">
+        <ul class = "nav-menu">
+          <li class = "nav-item"><a href = "#headline">Portfolio</a></li>
```
Comment (muneebulrehman) [inline index.html]: In the mobile menu, when I click on the portfolio link, it doesn't take me to the portfolio section but to the top of the page. Consider fixing it.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @spydaspider , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR
- Good commit messages

## Required Changes ♻️

- [ ] Kindly add the linter files for javascript.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by spydaspider** [comment]
Thank you.

**Review by mwanawabangona — APPROVED** [review]
Hi @spydaspider @Moathal,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by spydaspider** [comment]
Thank you! You guys have been of great help.

**Comment by spydaspider** [comment]
Sir please I already have the eslinttrc.json for the Javascript in my
project. Please I want to know if there is another file I will add to that?

**Pull request merged**